pkg/master/master_test.go

```diff
@@ -526,7 +526,7 @@ func testInstallThirdPartyAPIListVersion(t *testing.T, version string) {
 			setupEtcdList(fakeClient, "/ThirdPartyResourceData/company.com/foos/default", test.items)
 		}
 
-		resp, err := http.Get(server.URL + "/apis/company.com/" + version + "/namespaces/default/foos")
+		resp, err := http.Get(server.URL + "/apis/company.com/group/" + version + "/namespaces/default/foos")
```

Comment:
+1-- company.com IS the group, and if you want multiple groups from the same company, DNS subdomains can give you infinitely many.